doc/source/io.rst

```diff
@@ -1994,6 +1994,18 @@ into a flat table.
 .. ipython:: python
 
    from pandas.io.json import json_normalize
+   data = [
+             {'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
+             {'name': {'given': 'Mose', 'family': 'Regner'}},
+             {'id': 2, 'name': 'Faye Raker'}
+          ]
+   json_normalize(data)
+
+.. _io.jsonl:
+
+.. ipython:: python
+
+   from pandas.io.json import json_normalize
```

Comment:
FYI, this is how it looks on my local doc: 

<img width="824" alt="screen shot 2017-05-22 at 4 44 38 pm" src="https://cloud.githubusercontent.com/assets/28869596/26332788/0ccc76e8-3f0e-11e7-9679-985dc6a0aa3c.png">